docker/Dockerfile

```diff
@@ -99,6 +101,7 @@ RUN install2.r --repo "https://vimc.github.io/drat" \
 
 RUN Rscript -e 'remotes::install_github("vimc/vimpact")'
 RUN Rscript -e 'remotes::install_github("jokergoo/ComplexHeatmap")'
+RUN Rscript -e 'remotes::install_github("mrc-ide/beers")'
```

Comment:
This is in the vimc drat, so you should be able to install it as one of packages line 98